apis/rds/v1alpha1/zz_db_parameter_group.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by ack-generate. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// DBParameterGroupParameters defines the desired state of DBParameterGroup
+type DBParameterGroupParameters struct {
+	// Region is which region the DBParameterGroup will be created.
+	// +kubebuilder:validation:Required
+	Region string `json:"region"`
+	// The DB parameter group family name. A DB parameter group can be associated
+	// with one and only one DB parameter group family, and can be applied only
+	// to a DB instance running a database engine and engine version compatible
+	// with that DB parameter group family.
+	//
+	// To list all of the available parameter group families, use the following
+	// command:
+	//
+	// aws rds describe-db-engine-versions --query "DBEngineVersions[].DBParameterGroupFamily"
+	//
+	// The output contains duplicates.
+	// +kubebuilder:validation:Required
+	DBParameterGroupFamily *string `json:"dbParameterGroupFamily"`
+	// The name of the DB parameter group.
+	//
+	// Constraints:
+	//
+	//    * Must be 1 to 255 letters, numbers, or hyphens.
+	//
+	//    * First character must be a letter
+	//
+	//    * Can't end with a hyphen or contain two consecutive hyphens
+	//
+	// This value is stored as a lowercase string.
+	// +kubebuilder:validation:Required
+	DBParameterGroupName *string `json:"dbParameterGroupName"`
```

Comment:
We are using external name annotation for this field, so we should ignore it in `generator-config.yaml` like [this](https://github.com/crossplane/provider-aws/blob/7273d40/apis/apigatewayv2/v1alpha1/generator-config.yaml#L4). It needs to be ignored in all AWS calls in `zz_controller.go`, i.e. `CreateDBParameterGroupInput.DBParameterGroupName` etc.

Reply:
You need to re-run the code generator so that this field is removed.